lib/logstash/outputs/elastic_app_search.rb

```diff
@@ -44,6 +45,12 @@ class LogStash::Outputs::ElasticAppSearch < LogStash::Outputs::Base
   ENGINE_WITH_SPRINTF_REGEX = /^.*%\{.+\}.*$/.freeze
 
   def register
+    log_message = "The App Search product is deprecated and excluded from the version 9 of the Elastic Stack. " +
+      "This plugin is deprecated and will only receive security updates and critical bug fixes. " +
+      "We recommend transitioning to our native Elasticsearch tools. " +
+      "For more details, please visit https://www.elastic.co/guide/en/enterprise-search/current/app-search-workplace-search.html"
```

Comment:
@flexitrev same question here about the link https://www.elastic.co/guide/en/enterprise-search/current/app-search-workplace-search.html 
The instruction is a bit vague to me